components/gantry/oneaxis/oneaxis.go

```diff
@@ -23,6 +23,9 @@ import (
 
 var model = resource.DefaultModelFamily.WithModel("oneaxis")
 
+// LimitErrorMargin is added or subtracted from the location of the limit switch to ensure the switch is not passed.
+const LimitErrorMargin = 0.25
```

Comment:
make it unexported please 
```suggestion
const limitErrorMargin = 0.25
```